pkg/gotohelm/transpiler.go

```diff
@@ -129,6 +158,12 @@ func (t *Transpiler) transpileFile(f *ast.File) *File {
 		name = fileDirectives["filename"]
 	}
 
+	// Every file name is prepended with namespace
```

Comment:
Why has this change been added? If this is to handle name conflicts when we get to transpiling subcharts, could we instead just make subdirectories for dependent charts? IMO this makes it a lot more difficult to sort through the generated files.

The following seems like a much more ergonomic layout to me. Is there a reason we can't do that?
```
templates/
   deployment.yaml
   _deployment.tpl
   console/
       deployment.yaml
       _deployment.tpl
```

Reply:
agree I can change that